syft/frameworks/torch/hook/hook.py

```diff
@@ -714,8 +736,20 @@ def module_encrypt_(nn_self, **kwargs):
             if module_is_missing_grad(nn_self):
                 create_grad_objects(nn_self)
 
-            for element_iter in tensor_iterator(nn_self):
-                for p in element_iter():
+            for element_iter in named_tensor_iterator(nn_self):
+                for name_p, p in element_iter():
+                    if "running_var" in name_p:
```

Comment:
NITPICK: Maybe it would look cleaner to have if like:
```
if "running_var" not in name_p:
    continue

if nn_self.training:
   continue

warnings.warn(...)

````
^^This would be only to don't have so many nested "for, ifs"

Reply:
hmmm yeah but I need to execute `p.encrypt(inplace=True, **kwargs)`  for all elements of the loop and _after_ the transformation